### ISSUE TITLE
LanguageTool builder fails with exit code 141

### ISSUE THREAD

**Issue by Occhioverde** [NixOS/nixpkgs#197785]
### Steps To Reproduce
1. Add `languagetool.enable = true;` to your system module's `services`
2. Rebuild your system
3. See the error in log

### Build log
```console
root@Ricky-First-Laptop /e/nixos# nixos-rebuild switch
building the system configuration...
error: builder for '/nix/store/1cvd7xgpdc94kkwxk2vqdbr37snhacic-languagetool.conf.drv' failed with exit code 141
error: 1 dependencies of derivation '/nix/store/dsrvvhrhalr8li1i9vq0r2xmb52ywcnk-unit-languagetool.service.drv' failed to build
error: 1 dependencies of derivation '/nix/store/9w66lswkz6bxh6h06vfaf59clvc9ysmy-system-units.drv' failed to build
error: 1 dependencies of derivation '/nix/store/3z6f7snlqnwv4xc1ribq8kfwhw91sfzj-etc.drv' failed to build
error: 1 dependencies of derivation '/nix/store/b56bdlln9b5s30qyz8qk0a4x4pk51yl4-nixos-system-Ricky-First-Laptop-22.11.20221025.f994293.drv' failed to build
```

### Additional context
The Nix Flake is locked on Nixpkgs Unstable rev. f994293d1eb8812f032e8919e10a594567cf6ef7

### Notify maintainers
@edwtjo
@fedeinthemix

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
root@Ricky-First-Laptop /e/nixos# nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 5.19.12-zen1, NixOS, 22.11 (Raccoon), 22.11.20221013.ba187fb`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - channels(root): `"nixos-22.05"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by fedeinthemix** [comment]
It builds successfully on my system. If you have a git checkout could you try to run
```
cd path-to-your-nixos-checkout/nixos/tests
nix-build languagetool.nix 
```
This builds a VM with `LanguageTool` and checks it.

**Comment by Occhioverde** [comment]
> It builds successfully on my system

Yes, thanks to a comment on another issue I noticed that some paths in my Nix Store went corrupted as a result of an unexpected shutdown that occurred during a download, causing this error.

I managed to solve the problem by running `sudo nix-store --verify --check-contents --repair`, so I think that I can close this issue.

The next time I will check for corruptions before opening a ticket.

Thank you very much for your reply!
